tc-DAOlytics-operation/start.py

```diff
@@ -23,28 +26,72 @@ class AnalyzerHandler(BaseHTTPRequestHandler):
     host = os.getenv("DB_HOST")
     port = os.getenv("DB_PORT")
 
+    neo4j_dbName = os.getenv("NEO4J_DB")
```

Comment:
please update `.env.analyzer.proc.sample` in development and production directory!!

Reply:
Should I add `BACKEND_API_KEY` and `NOTIFY_BACKEND_URL` to development and production file (.env.analyzer.proc.sample)?